v2/options.go

```diff
@@ -25,6 +28,8 @@ type Options struct {
 	IndexPadding           uint64
 	IndexCodec             multicodec.Code
 	ZeroLengthSectionAsEOF bool
+	MaxIndexCidSize        uint64
```

Comment:
`Size` vs `Length` in the same optionset strike me as a bit odd. No hard preference, just raising it...

Reply:
Thanks for pointing this out. `ZeroLength*` is the odd one out; the rest of the codebase uses `Size` consistently. Will leave this as is and consider renaming on a separate PR